server/views/match-manual.njk

```diff
@@ -1,150 +1 @@
                        classes: "govuk-input--width-10",
                        id: "crn",
                        name: "crn",
                        pattern: "^[A-Za-z][0-9]{6}$",
                        errorMessage: errorMessage,
                        value: session.crn
                    }) }}

                    <button type="submit" class="govuk-button">Find record</button>

                    <p class="govuk-body"><a href="{{ useBackLink }}" class="govuk-link govuk-link--no-visited-state">Cancel</a></p>
                </form>
            {% else %}
                <form name="matchDefendant" id="matchDefendant" action="/{{ params.courtCode }}/match/defendant/{{ params.caseNo }}/confirm" method="post" novalidate>

                    <h3 class="govuk-heading-m">NDelius record found</h3>

                    <dl class="govuk-summary-list">
                        <div class="govuk-summary-list__row">
                            <dt class="govuk-summary-list__key">Name</dt>
                            <dd class="govuk-summary-list__value">{{ details.forename }} {{ details.surname }}</dd>
                        </div>
                        <div class="govuk-summary-list__row">
                            <dt class="govuk-summary-list__key">Date of birth</dt>
                            <dd class="govuk-summary-list__value">{{ params.moment(details.dateOfBirth, 'YYYY-MM-DD').format('D MMMM YYYY') }}</dd>
                        </div>
                        <div class="govuk-summary-list__row">
                            <dt class="govuk-summary-list__key">CRN</dt>
                            <dd class="govuk-summary-list__value">{{ details.otherIds.crn }}</dd>
                        </div>
                        <div class="govuk-summary-list__row">
                            <dt class="govuk-summary-list__key">PNC</dt>
                            <dd class="govuk-summary-list__value">{{ details.otherIds.pncNumber }}</dd>
                        </div>
                        <div class="govuk-summary-list__row">
                            <dt class="govuk-summary-list__key">Probation status</dt>
                            <dd class="govuk-summary-list__value">{{ details.probationStatus | string | capitalize if details.probationStatus }}</dd>
                        </div>
                    </dl>

                    <input type="hidden" id="crn" name="crn" value="{{ params.crn }}"/>
                    <button type="submit" class="govuk-button">Link record to defendant</button>

                    <p class="govuk-body">If this isn't the record you're looking for, you can
                        <a href="{{ useBackLink }}"
                           class="govuk-link govuk-link--no-visited-state">search again</a>.
                    </p>
                </form>
            {% endif %}

        </div>
        <div class="govuk-grid-column-one-third">&nbsp;</div>
    </div>

{% endblock %}


```

Comment:
Can you fix the formatting of this file please? Looks like somehow it's now minified or at least all on one line :/